src/modules/fossil_branch.rs

```diff
@@ -0,0 +1,215 @@
+use super::{Context, Module, ModuleConfig};
+
+use crate::configs::fossil_branch::FossilBranchConfig;
+use crate::formatter::StringFormatter;
+use crate::modules::utils::truncate::truncate_text;
+
+/// Creates a module with the Fossil branch of the check-out in the current directory
+///
+/// Will display the branch name if the current directory is a Fossil check-out
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let is_checkout = context
+        .try_begin_scan()?
+        .set_files(&[".fslckout"])
+        .is_match();
+
+    if !is_checkout {
+        return None;
+    }
+
```

Comment:
Please move this after the `config.disabled` check, because it should be cheaper.

Reply:
Changed the order of the checks now. This should probably be changed in the `pijul_channel` module as well to keep consistency between the modules.